edx_proctoring/admin.py

```diff
@@ -485,7 +486,9 @@ def save_model(self, request, obj, form, change):
             if change:
                 update_attempt_status(obj.proctored_exam.id, obj.user.id, form.cleaned_data['status'])
         except (ProctoredExamIllegalStatusTransition, StudentExamAttemptDoesNotExistsException) as ex:
-            messages.error(request, ex.message)
+            # prevent showing success message inappropriately
+            messages.set_level(request, messages.ERROR)
```

Comment:
Hi, @MichaelRoytman could you please clarify what success messages could be shown inappropriately? 
Maybe you have an example. I'd like to leave as much info to the user as we can, or I missed something?

Reply:
Hi, @idegtiarov. Thanks for the feedback! I'll attach a photo below of what is happening if I don't set the level of the messages. The model isn't saved due to an illegal status transition (the red banner), but the admin page is incorrectly informing the user that the model was saved successfully (the green banner). What do you think?

![image](https://user-images.githubusercontent.com/11871801/107526781-31c95580-6b86-11eb-8d5d-3dc497dbc8bc.png)

Reply:
Thanks for the clarification. You are right two opposite messages looks odd